SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -72,6 +72,10 @@ public enum PrivacyFeature: String {
     case setAsDefaultAndAddToDock
     case contentScopeExperiments
     case extendedOnboarding
+    case macOSBrowserConfig
+    case iOSBrowserConfig
+    // Demonstrative case for default value. Remove once a real-world feature is added
+    case intentionallyLocalOnlyFeatureForTests
```

Comment:
I'm not sure we will be removing that, as the way I understand this, is that in many cases fail-safe flags will be living for a certain period of time around the release and then be removed.